examples/assets_pandas_pyspark/assets_pandas_pyspark_tests/test_assets.py

```diff
@@ -1,15 +1,27 @@
-from assets_pandas_pyspark.assets.spark_weather_assets import spark_weather_assets
-from assets_pandas_pyspark.assets.weather_assets import weather_assets
+from assets_pandas_pyspark.assets import spark_asset, table_assets
 
-from dagster import materialize
+from dagster import load_assets_from_modules, materialize, with_resources
 from dagster._core.test_utils import instance_for_test
 
 
 def test_weather_assets():
+    from assets_pandas_pyspark.local_filesystem_io_manager import local_filesystem_io_manager
+
+    weather_assets = with_resources(
```

Comment:
Not a big deal, but could pass the resources directly to `materialize` instead of using `with_resources`. Or pull the resources assets from the Definitions object.